### ISSUE TITLE
Dropdown arrow on panel indicator conflicts with "Remove dropdown arrows" shell extension

### ISSUE THREAD

**Issue by errclman** [codito/gnome-pomodoro#228]
Application indicator in the top-panel conflicts(maybe?) with "remove dropdown arrows" extension and randomly appears and disappears when i restart gnome shell

![2016-07-04 01-11-43](https://cloud.githubusercontent.com/assets/13776349/16546687/665783c6-418d-11e6-9a5f-4eb041282df1.png)
![2016-07-04 01-14-50](https://cloud.githubusercontent.com/assets/13776349/16546688/6657fea0-418d-11e6-8c09-5fd268034d5e.png)

**Comment by kamilprusko** [comment]
Hi, sorry for late reply.

It's patched. Though I didn't experienced it when restarting gnome-shell, only when coming back from lockscreen... try it and feel free to reopen the issue.

**Comment by errclman** [comment]
Thx! It seems, that it fixed.

**Comment by errclman** [comment]
I used Pomodoro Indicator for a long time, so i found the way to reproduce this bug again. When i lock gnome3 session by clicking "Lock" or pressing Super+L and then unlock the session, arrow appears and only extension or shell restart hides arrow. I found a workaround by completely removing arrow from indicator.js

I removed this lines from indicator.js, so icon no more shows
`511:         this._arrow = PopupMenu.arrowIcon(St.Side.BOTTOM);`
`517:         this._hbox.add_actor(this._arrow);`

_p.s. sorry for my english_
